mass_mailing_switzerland/models/mass_mailing.py

```diff
@@ -28,6 +29,14 @@ class MassMailing(models.Model):
         compute='compute_events', store=True)
     unsub_event_ids = fields.Many2many(
         'mail.tracking.event', compute='compute_events')
+    mailing_origin = fields.Many2one(
```

Comment:
By convention in odoo, Many2one fields en with `_id` suffix (and x2Many fields with `_ids`). You can rename it to `mailing_origin_id` and update all references in code and in the view.